Package.swift

```diff
@@ -87,6 +87,26 @@ if ProcessInfo.processInfo.environment["SWIFTCI_INSTALL_RPATH_OS"] == "android"
  */
 let autoProducts = [swiftPMProduct, swiftPMDataModelProduct]
 
+let shouldUseSwiftBuildPackageDependency = (
```

Comment:
chore (blocking): Need to ensure this does not produce https://github.com/swiftlang/swift-package-manager/pull/8437#issuecomment-2773533124

```
.../swiftpm/Sources/Basics/Archiver/TarArchiver.swift:15:15: error: no such module 'TSCBasic'
```

Reply:
@drodriguez : Could you please try this change to ensure it does not break your workflow?

Reply:
This breaks the builds for people that chosen to use `SWIFTPM_NO_SWBUILD_DEPENDENCY` and avoid swift-build. After this change it seems that the swift-build dependency is required. Is that one of the intents of the change? The summary seems to talk about changing things when the swift-build framework is used, but nothing about requiring swift-build for every build.

Reply:
SwiftBuild is intended to be made available via the `--build-system swiftbuild` option.